client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html

```diff
@@ -38,6 +38,24 @@
                                 data-ng-bind="server.commercialRange !== 'housing' ? server.description || '-' : ('server_configuration_description_housing' | translate)"></dd>
                         </dl>
                     </div>
+                    <div class="oui-tile__item"
+                        data-ng-if="isEligibleForUpgrade() && upgradeName">
+                        <dl class="oui-tile__definition">
+                            <dt class="oui-tile__term">
+                                <span data-translate="server_configuration_upgrade_title"></span>
+                                <span class="badge" data-translate="server_configuration_upgrade_badge" aria-hidden="true"></span>
```

Comment:
Could it make sense to leave the information for screen readers ? It's not an icon, it adds information

Reply:
Yes, you're right. I'll update it